GNUmakefile

```diff
@@ -74,7 +75,9 @@ tools:  ## Install dev tools
 	go install github.com/terraform-linters/tflint@v0.49.0
 	go install github.com/rhysd/actionlint/cmd/actionlint@latest
 	go install golang.org/x/tools/go/analysis/passes/fieldalignment/cmd/fieldalignment@latest
+	go install github.com/vektra/mockery/v2@$(MOCKERY_VERSION)
```

Comment:
won't this pollute the go.mod file?

Reply:
cool, double checking sometimes when doing go install and not using latest things get added to the go.mod